addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -2489,6 +2496,18 @@ const Many2oneUserValueWidget = SelectUserValueWidget.extend({
     // Private
     //--------------------------------------------------------------------------
 
+    /**
+     * Caches the rpc.
+     *
+     * @override
+     */
+    async _rpc() {
+        const cacheId = JSON.stringify(...arguments);
+        if (!this._rpcCache[cacheId]) {
+            this._rpcCache[cacheId] = await this._super(...arguments);
+        }
+        return this._rpcCache[cacheId];
+    },
```

Comment:
Why not :+1: However, it would be better to not await in this case: just save the promise returned by the super call in the cache and return it. That way you return the exact same thing as super would and still can handle the case where the promise is rejected.